### PR TITLE
Feat: Added Badge Component

### PR DESCRIPTION
**Pull request by AshwiniSetty**
Added Badge component

Browse the Preview here: 
https://deploy-preview-8--miniwii-ui.netlify.app/components/badges/badges

### CODE REVIEW

File: components/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+.status-invisible {
+    background-color: var(--bg-color);
+    border: 3.5px solid var(--secondary-color);
+}
+
+.badge-status.badge-notification {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (dev-enforced) [inline components/badges/badges.css]: You can make a separate utility class for this named as minwii-flex-column-centered and pass these properties inside it and use it inside wherever required.

**Review by dev-enforced — APPROVED** [review]
The structure looks good and have given a minor suggestion. The live preview also looks good and well documented: https://deploy-preview-8--miniwii-ui.netlify.app/components/badges/badges
Good work @AshwiniSetty

File: components/badges/badges.css [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    position: absolute;
+    border: none;
+    left: 60%;
+    bottom: 65%;
+    font-size: 0.9rem;
```
Comment (veereshbv04) [inline components/badges/badges.css]: If anything is less than 1rem then use pixel units

File: components/badges/badges.css [hunk 3]
```diff
@@ -0,0 +1,65 @@
+.profile-badge {
+    position: absolute;
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.5rem;
```
Comment (veereshbv04) [inline components/badges/badges.css]: same here use px for anything less than 1 rem

File: components/badges/badges.css (hunk 2 above)
Comment (AshwiniSetty) [reply components/badges/badges.css]: will change that to px

File: components/badges/badges.css (hunk 3 above)
Comment (AshwiniSetty) [reply components/badges/badges.css]: will change that to px

File: components/badges/badges.css (hunk 1 above)
Comment (AshwiniSetty) [reply components/badges/badges.css]: thanks for that suggestion, will do that.

**Comment by jyothishp994** [comment]
Minwii is a great component library with very well documented. I have a suggestion on the sidebar. Sidebar seems to be hidden when window is less than 800px, Rather than doing that, You can add flex-direction column . As the doc-container has two child, which is sidebar and content, This would display the sidebar and content vertically in case of smaller screen sizes
`@media screen and (max-width: 800px)
.side-bar {
    display: none;
}
`
`@media screen and (max-width: 800px)
.doc-container{
    flex-direction:column
}
`

**Review by jyothishp994 — COMMENTED** [review]
Code is clean and well indented.

**Pull request merged**